### DISCUSSION TITLE
My Dashboard / Display

### DISCUSSION THREAD

**Discussion by BJReplay** [jasonacox/Powerwall-Dashboard#161] [Show and tell]
I'm pretty pleased with this.

Thanks to @jasonacox for the Powerwall Dashboard, @mcbirse for the Tesla History import tool and Weather History import tool that were the inspiration.

I had a weather station where the internal display died.  Instead of buying a new one, I re-purposed an original Surface RT (runs a 32-bit NVIDIA Tegra ARM chipset) as a display.

Whilst the built in version of internet explorer won't actually display Grafana, it has an RDP client.

I have two dashboards set up that display reasonably on the 1366 by 768 screen.  Screen shots below.  They're pretty busy, but at the same time provide both at a glance info, as well as details.

I've set them up as a Grafana Playlist with a 15 second cycle, in TV mode, and then I've set up a ubuntu desktop using https://docs.linuxserver.io/images/docker-rdesktop running on the same docker desktop that is hosting Powerwall-Dashboard, and I RDP into that from the Surface RT.

Short video at the end showing the transition from one dashboard to the next.

![image](https://user-images.githubusercontent.com/37993507/213060795-70e35bea-0957-4c8b-a872-44c7e3809882.png)
![image](https://user-images.githubusercontent.com/37993507/213060821-cddc6c15-f3cf-4739-8cd4-1ce818274cd4.png)

https://user-images.githubusercontent.com/37993507/213060647-2133023b-2889-4612-9fbc-214752044fee.mp4

**Comment by jasonacox** [comment]
Wow @BJReplay  - that's beautiful! 👍

**Reply by mcbirse** [reply]
That looks fantastic - well done excellent work!

Nice to know the history tools have been useful for re-purposing and adding features for both Powerwall-Dashboard and pyPowerwall now - that's great. 😊
